nilearn/datasets/struct.py

```diff
@@ -735,18 +742,27 @@ def fetch_oasis_vbm(n_subjects=None, dartel_version=True, data_dir=None,
     data_usage_agreement = files[-1]
 
     # Keep CSV information only for selected subjects
-    csv_data = np.recfromcsv(ext_vars_file)
+    csv_data = pd.read_csv(ext_vars_file)
     # Comparisons to recfromcsv data must be bytes.
     actual_subjects_ids = [("OAS1" +
                             str.split(os.path.basename(x),
-                                      "OAS1")[1][:9]).encode()
+                                      "OAS1")[1][:9])
                            for x in gm_maps]
     subject_mask = np.asarray([subject_id in actual_subjects_ids
-                               for subject_id in csv_data['id']])
+                               for subject_id in csv_data['ID']])
     csv_data = csv_data[subject_mask]
-
+    csv_data = csv_data.rename(
+        columns={c: c.lower() for c in csv_data.columns}
+    )
+    csv_data = csv_data.rename(
+        columns={c: c.replace("/", "") for c in csv_data.columns}
```

Comment:
doesn't matter but it could be done in one go with `c.lower().replace("/", "")`

Reply:
Indeed, thanks! (see 55db108)